ph5/clients/tests/test_ph5tostationxml.py

```diff
@@ -223,8 +223,9 @@ def test_create_obs_network(self):
         with LogCapture() as log:
             log.setLevel(logging.WARNING)
             self.parser.create_obs_network()
+            errmsgs = sorted(self.errmsgs)
             for i in range(len(self.errmsgs)):
```

Comment:
```suggestion
```

Reply:
I don't understand. Why do you remove the for loop? Do you mean change lines from 226 to 228 to: 
```
            for i in range(len(self.errmsgs)):
                self.assertEqual(log.records[i].msg, sorted(errmsgs)[i])
```